stdlib/public/Concurrency/Task.cpp

```diff
@@ -355,3 +355,49 @@ void swift::swift_task_run(AsyncTask *taskToRun) {
 JobFlags swift::swift_task_getJobFlags(AsyncTask *task) {
   return task->Flags;
 }
+
+namespace {
+  
+/// Structure that gets filled in when a task is suspended by `withUnsafeContinuation`.
+struct AsyncContinuationContext {
+  // These fields are unnecessary for resuming a continuation.
+  void *Unused1;
+  void *Unused2;
+  // Pointer to where to store an error result.
+  SwiftError **ErrorResult;
```

Comment:
The error pointer is stored directly. This should be a `SwiftError *ErrorResult`.